presubmit.sh

```diff
@@ -0,0 +1,15 @@
+set -e
+set -x
+
```

Comment:
Could we make Travis use this script as well, instead of (not quite exactly) duplicating this at .travis.yml?